Library/Homebrew/cmd/install.rb

```diff
@@ -197,10 +197,10 @@ def self.install
     begin
       formulae, casks = args.named.to_formulae_and_casks
                             .partition { |formula_or_cask| formula_or_cask.is_a?(Formula) }
-    rescue FormulaOrCaskUnavailableError, Cask::CaskUnavailableError => e
+    rescue FormulaOrCaskUnavailableError, Cask::CaskUnavailableError
       retry if Tap.install_default_cask_tap_if_necessary(force: args.cask?)
 
-      raise e
+      raise
```

Comment:
This is needed to avoid a typechecking error.

```
/u/l/Homebrew (warn-on-cask-install-when-already-installed|✔) $ brew tc
cmd/install.rb:240: Changing the type of a variable in a loop is not permitted https://srb.help/7001
     240 |      rescue Cask::CaskAlreadyInstalledError => e
                                                          ^
  Existing variable has type: NilClass
  Attempting to change type to: T.untyped

  Original type from:
    cmd/install.rb:198:
     198 |      formulae, casks = args.named.to_formulae_and_casks
     199 |                            .partition { |formula_or_cask| formula_or_cask.is_a?(Formula) }
     200 |    rescue FormulaOrCaskUnavailableError, Cask::CaskUnavailableError => e
     201 |      retry if Tap.install_default_cask_tap_if_necessary(force: args.cask?)
     202 |
     203 |      raise e
    cmd/install.rb:168: Possibly uninitialized (NilClass) in:
     168 |  def self.install
            ^^^^^^^^^^^^^^^^
Errors: 1
Check https://docs.brew.sh/Typechecking for more information on how to resolve these errors.
```

It should behave same way though.